worlds/alttp/Options.py

```diff
@@ -79,41 +80,44 @@ def hints_useful(self):
 
 
 class bigkey_shuffle(DungeonItem):
-    """Big Key Placement"""
+    """How Big Keys will be placed."""
     item_name_group = "Big Keys"
     display_name = "Big Key Shuffle"
 
 
 class smallkey_shuffle(DungeonItem):
-    """Small Key Placement"""
+    """How Small Keys will be placed."""
     option_universal = 5
     item_name_group = "Small Keys"
     display_name = "Small Key Shuffle"
 
 
 class compass_shuffle(DungeonItem):
-    """Compass Placement"""
+    """How Compasses will be placed."""
     item_name_group = "Compasses"
     display_name = "Compass Shuffle"
 
 
 class map_shuffle(DungeonItem):
-    """Map Placement"""
+    """How maps will be placed."""
     item_name_group = "Maps"
     display_name = "Map Shuffle"
 
 
 class Crystals(Range):
```

Comment:
wondering if this should be named CrystalsRange to sound more generic

Reply:
i mean it could be? but it's only used for 2 options here so eh